test/micro/case_test.rb

```diff
@@ -150,4 +150,106 @@ def test_inspect
       Multiply.inspect
     )
   end
+
+  class Bomb < Micro::Case
+    attributes :defused
+
+    def call!
+      Try do
+        if defused
+          :yay
+        else
+          raise 'Boooom!'
+        end
+      end
+    end
+  end
+
+  def test_try
+    result = Bomb.call(defused: true)
+
+    assert_success_result(result, value: { try: :yay }, type: :ok)
+
+    result = Bomb.call(defused: false)
+
+    assert_failure_result(result, type: :exception)
+    assert result.value[:try].is_a? RuntimeError
```

Comment:
@serradura I think there's a better way to check this, so I'll wait for your feedback.

Reply:
```suggestion
    assert_failure_result(result, type: :exception)
    assert_kind_of(RuntimeError, result[:try])
```